internal/local/structure_v2_test.go

```diff
@@ -44,15 +44,27 @@ func TestParseDataSources(t *testing.T) {
 
 	testRoot := filepath.Join(wd, "testdata/data_sources")
 
+	mdbAtlasStr := "mongodb-atlas"
 	t.Run("should return the parsed data sources directory with nested rules and schema", func(t *testing.T) {
 		dataSources, err := parseDataSources(testRoot)
 		assert.Nil(t, err)
-		assert.Equal(t, []DataSourceStructure{{
+		assert.Match(t, []DataSourceStructure{{
```

Comment:
[q] why the switch from `Equal` to `Match`?

Reply:
I had encountered a type match error earlier when testing but that seems to be resolved, will change it back!